python/lsst/ip/isr/isrTask.py

```diff
@@ -510,9 +521,9 @@ def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, def
                 self.overscanCorrection(ccdExposure, amp)
 
         if self.config.doCrosstalk:
-            # DECam requires a dataRef for crosstalk correction
-            if otherDataRef:
-                self.crosstalk.run(ccdExposure, otherDataRef)
+            if crosstalkSources is not None:
+                # crosstalkSources is used by DECam
+                self.crosstalk.run(ccdExposure, crosstalkSources)
             else:
                 self.crosstalk.run(ccdExposure)
```

Comment:
I think you should add the crosstalkSources to the default `CrosstalkTask`.  It can default to `None` in the base class.

Reply:
Think I've got this now.